src/main/webapp/projectContainer/Submissions.js

```diff
@@ -1,47 +1,153 @@
+import { MultiTab } from "../components/MultiTab";
+import { Table } from "../components/Table";
+import { Files } from "../util/ajax";
+import './Submissions.css';
+
+const headers =
+  [
+    { name: 'Number', value: 'number' },
+    { name: 'Description', value: 'comments' },
+    { name: 'Documents', value: 'documents' },
+    { name: 'Created', value: 'createDate' },
+  ];
+
 
 export const Submissions = hh(class Submissions extends Component {
 
   constructor(props) {
     super(props);
     this.state = {
-      content: ''
+      amendments:[],
+      others: [],
+      amendmentDocuments: [],
+      otherDocuments: [],
     };
   }
 
   componentDidMount() {
-    this.getSubmissions();
+    this.getDisplaySubmissions();
   }
 
-  getSubmissions() {
-    ProjectMigration.getSubmissions(component.serverURL, component.projectKey).then(resp => {
+  getDisplaySubmissions = () => {
+    let amendments = {};
+    let others = {};
+    ProjectMigration.getDisplaySubmissions(component.serverURL, component.projectKey).then(resp => {
+      amendments = resp.data.groupedSubmissions.Amendment;
+      others = resp.data.groupedSubmissions.Other;
+
+      amendments.forEach(other=> {
+        other.documents.forEach(document => {
+          Files.getDocument(document.id).then(resp => {
+            document.document = resp.data.document;
+          });
+        });
+      });
+
+      others.forEach(other=> {
+        other.documents.forEach(document => {
+          Files.getDocument(document.id).then(resp => {
+            document.document = resp.data.document;
+          });
+        });
+      });
+
       this.setState(prev => {
-        prev.content = resp.data;
+        prev.amendments = amendments;
+        prev.others = others;
         return prev;
-      }, () => {
-        this.loadSubmissions();
-      })
+      });
     });
   };
 
-  loadSubmissions() {
-    $(".submissionTable").DataTable({
-      dom: '<"H"Tfr><"pull-right"B><div>t</div><"F"lp>',
-      buttons: [],
-      language: { search: 'Filter:' },
-      pagingType: "full_numbers",
-      pageLength: 50,
-      columnDefs: [{ targets: [1, 2], orderable: false }],
-      order: [0, "desc"]
-    });
-    $("#submission-tabs").tabs();
+
+  getDocumentLink = (data) => {
+    return [component.serverURL, 'api/files-helper/get-document?id=' + data].join("/");
+  };
+
+  submissionEdit = (data) => {
+    const indexButton = a({
+      className: 'btn btn-default btn-xs pull-left link-btn',
+      href: `${component.contextPath}/submission/index?projectKey=${component.projectKey}&submissionId=${data.id}`
+    }, [component.isAdmin === true ? 'Edit': 'View']);
+    const submissionComment = span({className: 'submission-comment'}, [data.comments]);
+    return h(Fragment, {}, [indexButton, submissionComment]);
+  };
+
+  redirectNewSubmission(e) {
+    window.location.href=`${component.serverURL}/api/submissions/add-new?projectKey=${component.projectKey}&type=${e.target.id}`;
```

Comment:
Do we really have to redirect here? We're trying very hard to navigate to places instead of redirect. This pattern breaks the SPA model.